spec/fixtures/expired_cert.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
It would be nice to have a expired_cert.pem.to_create.txt file that has the commands used to generate the pem cert.

Reply:
This involved changing the system time on my workstation, I couldn't get any alternative to work.  Not the sort of thing you want to automate.  This will get solved when we include a lib for CA/cert/key generation in a subsequent PR.